python/ray/air/tests/test_data_batch_conversion.py

```diff
@@ -1,25 +1,105 @@
 import pytest
+import unittest
 
 import pandas as pd
 import numpy as np
 import pyarrow as pa
 
 from ray.air.constants import TENSOR_COLUMN_NAME
 from ray.air.util.data_batch_conversion import convert_batch_type_to_pandas
+from ray.air.util.data_batch_conversion import _convert_batch_type_to_numpy
 from ray.air.util.data_batch_conversion import convert_pandas_to_batch_type
 from ray.air.util.data_batch_conversion import DataType
 from ray.air.util.tensor_extensions.pandas import TensorArray
 from ray.air.util.tensor_extensions.arrow import ArrowTensorArray
 
 
-def test_pandas_pandas():
-    input_data = pd.DataFrame({"x": [1, 2, 3]})
-    expected_output = input_data
-    actual_output = convert_batch_type_to_pandas(input_data)
-    pd.testing.assert_frame_equal(expected_output, actual_output)
+class BatchConversionTest(unittest.TestCase):
```

Comment:
The rest of the AIR tests are using pytest and not unittest- can we consolidate on that?

Reply:
i added unittest because i need `assertDictEqual`. I didn't see an equivalent in pytest tho but happy to use if there's one available.

Reply:
would `d1 == d2` work?

it seems that unit test `assertDictEqual` is only useful for additional diagnostics https://stackoverflow.com/questions/34414326/why-is-assertdictequal-needed-if-dicts-can-be-compared-by. 

Would prefer consistency in tests over the additional diagnostics that unittest provides